cnd/src/created_swap.rs

```diff
@@ -0,0 +1,25 @@
+pub use crate::swap_protocols::{hbit, herc20, LocalSwapId, Role};
+use libp2p::PeerId;
+
+/// Data required to create a swap.
+///
+/// 'create' a swap is defined as the process of initiating a swap within `cnd`.
+/// The data required to do so is assumed to have been negotiated between the
+/// two parties prior to each creating the swap and is passed into cnd via the
+/// REST API.
+#[derive(Debug, Clone, PartialEq)]
+pub struct CreatedSwap<A, B> {
+    /// Node specific swap identifier.
+    pub swap_id: LocalSwapId,
+    /// The parameters used on the alpha ledger.
+    pub alpha: A,
+    /// The parameters used on the beta ledger.
+    pub beta: B,
+    /// Peer ID of the swap counterparty.
+    pub peer: PeerId,
+    /// Role of the node in this swap, Alice or Bob.
+    pub role: Role,
+}
+
+pub type HbitHerc20CreatedSwap = CreatedSwap<hbit::CreatedSwap, herc20::CreatedSwap>;
```

Comment:
🙃 Not too keen on this. Trading less characters for more indirection and basically the same name is not worth it for me.

Reply:
ok, I was indecisive about theses aliases, will remove.